packages/wrangler/src/config/config.ts

```diff
@@ -168,6 +168,18 @@ export interface ConfigFields<Dev extends RawDevConfig> {
 				[key: string]: string;
 		  }
 		| undefined;
+
+	/**
+	 * By default, wrangler.toml is the source of truth for your environment configuration, like a terraform file.
+	 *
+	 * If you change your vars in the dashboard, wrangler *will* override/delete them on its next publish.
+	 *
+	 * If you want to keep your dashboard vars when wrangler deploys, set this field to true.
+	 *
+	 * @default false
+	 * @nonInheritable
+	 */
+	keep_vars?: boolean;
```

Comment:
Adding this type here implies that it is possible to set this via wrangler.toml.
Is that what you intend? If so then we need to add code to `validation.ts` to check it and copy if over; and also tests in `configuration.test.ts`.

Reply:
That's what I intended, yeah. I assume we should aim for args/config parity?

Reply:
We should